Improvement/element details layout

Grumblesmith opened the pull request:
Updating element details layout

Commit:
Fixes #17 - improves the element details dashboard layout

Commit:
Adding release history entry

rclagett reviewed (changes requested):
netuitive.aws.rds.overallutilization computed metric was removed with a previous release: https://github.com/netuitive-community-packages/netuitive-packages-aws-rds#version-152
Merge state

Commit:
Removing netuitive.aws.rds.overallutilization from Details Dashboard
As @rclag1126 correctly pointed out,
netuitive.aws.rds.overallutilization was removed in the previous
version.

Grumblesmith commented:
@rclag1126 great catch! I fixed the release history and reverted the changes to the two Utilization widgets.
